src/DynamoCore/packages.config

```diff
@@ -2,7 +2,7 @@
 <packages>
   <package id="DynamoVisualProgramming.LibG_227_0_0" version="2.13.0.6656" targetFramework="net48" developmentDependency="true" />
   <package id="DynamoVisualProgramming.LibG_228_0_0" version="2.13.0.6656" targetFramework="net48" developmentDependency="true" />
-  <package id="Greg" version="2.1.7962.24801" targetFramework="net48" />
+  <package id="Greg" version="2.1.7972.24350" targetFramework="net48" />
   <package id="Newtonsoft.Json" version="8.0.3" targetFramework="net48" />
   <package id="RestSharp" version="106.12.0" targetFramework="net48" />
```

Comment:
@aparajit-pratap it looks like either version number is fine as long as the hint path only has a single 0.